srcpkgs/luarocks/template

```diff
@@ -1,38 +1,77 @@
 # Template file for 'luarocks'
 pkgname=luarocks
-version=2.4.4
+version=3.0.4
 revision=1
 archs=noarch
 build_style=configure
-configure_args="
- --prefix=/usr
- --sysconfdir=/etc/luarocks
- --lua-version=5.3
- --lua-suffix=5.3
- --with-lua-include=/usr/include/lua5.3
- --with-downloader=curl
+configure_args=" --prefix=/usr --sysconfdir=/etc
+ --lua-version=5.3 --with-lua-include=/usr/include/lua5.3
```

Comment:
I prefer the previous style for clarity, but whatever.

Reply:
reverted